src/ucs/config/parser.h

```diff
@@ -281,6 +284,10 @@ void ucs_config_help_generic(char *buf, size_t max, const void *arg);
                                     ucs_config_clone_double,     ucs_config_release_nop, \
                                     ucs_config_help_generic,     "time value: <number>[s|us|ms|ns]"}
 
+#define UCS_CONFIG_TYPE_TIME_LONG  {ucs_config_sscanf_time_long, ucs_config_sprintf_time_long, \
```

Comment:
good point to add this to ucs/config, maybe name it "UCS_CONFIG_TYPE_TIME_UNITS" (and also the respective functions)